hashutils/id.go

```diff
@@ -33,7 +35,10 @@ func GetLengthLimitedID(fixedPrefix, suffix string, maxLength int) string {
 		// start with the character that we use to denote a shortened string, which could
 		// result in a clash.  Hash the value and truncate...
 		hasher := sha256.New()
-		hasher.Write([]byte(suffix))
+		_, err := hasher.Write([]byte(suffix))
+		if err != nil {
+			log.Panic("failed to write hash suffix")
```

Comment:
Nit: this log message is a bit sloppy.  You're not writing a "hash suffix", you're writing the suffix to the hash.  Also, you should log the error and use sentence-case for log messages (since they stand alone in the output, unlike `error`s, which are intended to be logged as _part_ of a message:
```
log.WithError(err).Panic("Failed to write suffix to hash.")
```

Reply:
Fixed